assets/lokomotive-kubernetes/bare-metal/flatcar-linux/kubernetes/variables.tf

```diff
@@ -59,6 +59,12 @@ variable "clc_snippets" {
   default     = {}
 }
 
+variable "labels" {
+  type        = list(string)
+  description = "List of labels for worker nodes."
+  default     = []
+}
+
```

Comment:
Idem, why not use the same format on all platforms? Packet is using string (instead of list), this will just create more work when we want to unify the behaviour for no good reason. Or is there a reason?

Reply:
We take list of labels, we shouldn't treat a list as a string. We should be unifying the behavior towards correct formats, not the opposite IMO.

Reply:
Well, this series is not doing so. It is just creating inconsistencies. I'm fine if you prefer to have it this way and create an issue to change packet to this format with the "tech-debt" label.

Reply:
Created https://github.com/kinvolk/lokomotive/issues/489.

Reply:
Cool, thanks!